### ISSUE TITLE
Merging of SimCalorimeterHit

### ISSUE THREAD

**Issue by LinghuiWuIHEP** [cepc/CEPCSW#35]
Hi @mirguest @fucd ,

For the calorimeter, the steps of Geant4 simulation in the same cell should be merged before filling to SimCalorimeterHit. Where is the merging implemented?

Cheers,
Linghui

**Comment by qwert2333** [comment]
Hi, 
  I just found that in Sensitive Detector (DetSimSD) the contribution is stored as dd4hep::sim::Geant4Hit::Contribution, which only contains trackID, pdgID, deposit and time information from G4step. 
  So although edm4hep::SimCalorimeterHitContribution has member Position, we still lost the position of G4step. 
  Is that possible to change the EDM in DetSimSD? Otherwise I think we can not merge the SimHit.

Cheers,
Fangyi

ref: https://dd4hep.web.cern.ch/dd4hep/reference/structdd4hep_1_1sim_1_1Geant4Hit_1_1MonteCarloContrib.html
ps. I found a very similar class which contains position: 
https://dd4hep.web.cern.ch/dd4hep/reference/classdd4hep_1_1sim_1_1Geant4HitData_1_1MonteCarloContrib.html
